src/main/java/org/alliancegenome/curation_api/services/GeneDiseaseAnnotationService.java

```diff
@@ -92,18 +95,14 @@ public ObjectResponse<GeneDiseaseAnnotation> delete(Long id) {
 	public void removeOrDeprecateNonUpdated(String curie, String loadDescription) { }
 
 	public List<Long> getAnnotationIdsByDataProvider(BackendBulkDataProvider dataProvider) {
-		List<Long> annotationIds;
-
-		String sourceOrg = dataProvider.sourceOrganization;
-
-		if( StringUtils.equals(sourceOrg, "RGD") ){
-			annotationIds = geneDiseaseAnnotationDAO.findAllAnnotationIdsByDataProvider(dataProvider.sourceOrganization, dataProvider.canonicalTaxonCurie);
-		} else {
-			annotationIds = geneDiseaseAnnotationDAO.findAllAnnotationIdsByDataProvider(dataProvider.sourceOrganization);
-		}
-
-		annotationIds.removeIf(Objects::isNull);
-
+		Map<String, Object> params = new HashMap<>();
+		params.put("dataProvider.sourceOrganization.abbreviation", dataProvider.sourceOrganization);
```

Comment:
Given the amount that we are using this field name... might be good to put them into a constants file... it would just make it easier to change in the future if this field changes.

Reply:
@oblodgett Done